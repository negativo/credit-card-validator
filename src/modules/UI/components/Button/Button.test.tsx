import React from 'react';
import { Button } from './';
import renderer from 'react-test-renderer';

it('renders correctly without props', () => {
  const tree = renderer.create(<Button>Test Button</Button>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with props', () => {
  const tree = renderer
    .create(<Button disabled={true}>Test Button</Button>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
