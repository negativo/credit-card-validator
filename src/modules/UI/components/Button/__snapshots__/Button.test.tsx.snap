// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with props 1`] = `
<div
  className="sc-AxjAm gRUtaw"
  disabled={true}
>
  Test Button
</div>
`;

exports[`renders correctly without props 1`] = `
<div
  className="sc-AxjAm cLLNJw"
>
  Test Button
</div>
`;
